Core/src/org/sleuthkit/autopsy/coreutils/NetworkUtils.java

```diff
@@ -18,21 +18,25 @@
  */
 package org.sleuthkit.autopsy.coreutils;
 
+import java.io.IOException;
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.net.UnknownHostException;
-import java.util.StringTokenizer;
+import java.util.logging.Level;
```

Comment:
Oh, just noticed this. This is not the right logger, could be the coreutils one.

Reply:
This is the logging level, not logger. My mistake.